genesis-connect-woocommerce.php

```diff
@@ -36,12 +36,16 @@ function gencwooc_setup() {
 
 	require_once GCW_ADMIN_DIR . '/notices.php';
 	$ready = true;
-	
-	if ( ! in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {		
+
+	if ( ! function_exists( 'is_plugin_active' ) ) {
+		require_once( ABSPATH . '/wp-admin/includes/plugin.php' );
+	}
+
+	if ( ! is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
 		add_action( 'admin_notices', 'gencwooc_woocommerce_notice' );
 		$ready = false;
 	}
-	
+
 	if ( ! function_exists( 'genesis' ) ) {
```

Comment:
@marksabbath Do you think we should alter this to the following?

`if ( ! function_exists( 'genesis' ) &&  ! is_multisite() )`

At the moment sites on the multisite network not running Genesis themes will see this notice if they're using a non-Genesis theme and Genesis Connect is network-activated:

<img width="1112" alt="manage_themes_ _multisite_ _wordpress" src="https://user-images.githubusercontent.com/647669/42413613-37b4b94c-8224-11e8-97d3-435053004a59.png">

There might be a better solution for this that also prevents the plugin running further if multisite is active but Genesis is not.

Reply:
Hey, @nickcernis I liked the second approach. I think it's gonna fix the issue and keep things simple :D

I'll make this update and commit the code.